src/hotspot/share/jfr/periodic/sampling/jfrThreadSampling.cpp

```diff
@@ -159,9 +162,11 @@ static inline bool is_valid(const PcDesc* pc_desc) {
   return pc_desc != nullptr && pc_desc->scope_decode_offset() != DebugInformationRecorder::serialized_null;
 }
 
-static bool compute_top_frame(const JfrSampleRequest& request, frame& top_frame, bool& in_continuation, JavaThread* jt) {
+static bool compute_top_frame(const JfrSampleRequest& request, frame& top_frame, bool& in_continuation, JavaThread* jt, bool& biased) {
   assert(jt != nullptr, "invariant");
 
+  biased = false;
```

Comment:
Can be moved after the has_last_Java_frame() check.

Reply:
I removed the assignment.